backend/.env.example

```
@@ -11,7 +11,7 @@ ACCESS_TTL=                          #access token time in minitues ie.15
 EMAIL_ID= 
 CONTACT_EMAIL_ID= 
 RESEND_API_KEY=
-NODE_ENV=
+NODE_ENV=                  # keep this production for refresh token to work
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)